model/edgeless.ml

```diff
@@ -75,18 +78,19 @@ or more outputs.
 module Invocation : sig
   type t
 end = struct
-  type t = { alias : alias; func : Function.t; outputs : output list }
+  type t = { alias : alias; func : Function.t }
 end
 
 (** Workflows.
     
-A workflow is a named list of function invocations, representing a DAG.
+A workflow is a named list of function invocations, representing a single
+function, chain, DAG, or general graph.
 *)
 
 module Workflow : sig
   type t
 end = struct
-  type t = { alias : string; functions : Invocation.t list }
+  type t = { w_name : w_name; functions : Invocation.t list; output_mapping : output * alias list }
```

Comment:
Should add to the comment indicating what `output_mapping` represents

Reply:
Also I've remembered the "dictionary" syntax -- where I've been writing `'key * 'value list` instead write `'key * 'value Hashtbl.t` I think